src/esp/bindings/SensorBindings.cpp

```diff
@@ -188,8 +189,102 @@ void initSensorBindings(py::module& m) {
           R"(The distance to the near clipping plane for this CameraSensor uses.)")
       .def_property(
           "far_plane_dist", &CameraSensor::getFar, &CameraSensor::setFar,
-          R"(The distance to the far clipping plane for this CameraSensor uses.)");
-
+          R"(The distance to the far clipping plane for this CameraSensor uses.)")
```

Comment:
Given that we still need something on the python side for noise models, this can also be done in python.